README.md

```diff
@@ -182,10 +210,96 @@ Here the steps:
    * branch            main       -> FETCH_HEAD
    Successfully mreged and updated refs/heads/main.
    ```
+## Contributing through a branch
 
----
+The following explains the steps to [contribute directly to a repository using
+a branch](https://docs.github.com/en/github/getting-started-with-github/quickstart/github-flow).
 
-## Resources
+In a nutshell, here the steps:
+
+1. Clone the repository on your local computer
 
+   :bangbang: **NOTE** This is different from the above as you are cloning the original
+   repository and not your fork.
+   
+   Command:
+   ```shell
+   git clone https://github.com/snowex-hackweek/learning-git
+   ```
+
+1. Create a branch
+
+   Choose a unique name that is not already taken in the repository and helps
+   other collaborators to identify your work. In this case we are adding a branch
+   with name 'feature_xyz'
+   
+   ```shell
+   # The '-b' is indicating a new branch with the name `feature_xyz`
+   # It will also change to that.
+   git co -b feature_xyz
+   ```
+   
+   Sample output:
+   ```shell
+   Switched to a new branch 'feature_xyz'
+   ```
+   
+1. Verify you are on the new branch
+
+    ```shell
+    # Show all branches of the repository, the current active has the `*`
+    git branch
+    ```
+
+   Output:
+    ```shell
+    main
+    * feature_xyz
+    ```
+
+1. Make changes within the new branch
+
+   With this practice repository, you have to create a folder within
+   **`contributions`** folder with your github username as the folder name. Use
+   a new markdwon file (`.md`) or text file (`.txt`) file to practice git with.
```

Comment:
I'm wondering if we also should allow `.rst`? Some people might be familiar with that.

Reply:
I think the two options should cover most of participants. It is also a simple practice run and I like simple options :-)